experimental/Schemes/Resolution_structure.jl

```diff
@@ -1,54 +1,138 @@
-## Warnung: show auf desingMor geht noch nicht!!!
 export _desing_curve
 export  find_refinement_with_local_system_of_params
 
+##############################################################################
+## Concrete Type for normalization
+## very similar to CoveredSchemeMorphism, but allowing disjoint handling
+## of disjoint components
+##############################################################################
+@attributes mutable struct NormalizationMorphism{
+    DomainType<:AbsCoveredScheme,
+    CodomainType<:AbsCoveredScheme,
+    BaseMorphsimType
+   } <:AbsCoveredSchemeMorphism{
+                                 DomainType,
+                                 CodomainType,
+                                 Nothing,
+                                 NormalizationMorphism,
+                                }
+
+  X::DomainType
+  Y::CodomainType
+  f::CoveringMorphism
```

Comment:
```suggestion
  underlying_morphism::CoveredSchemeMorphism
```
It is just as @simonbrandhorst says. I will also suggest accompanying changes below.